setup.py

```diff
@@ -4,7 +4,7 @@
 from setuptools import setup, find_packages
 
 install_requires = [
-    'coalaip>=0.0.1.dev2',
+    'pycoalaip>=0.0.1.dev2',
```

Comment:
> Actually the package name is coalaip. Will revert.

doesn't matter for this one, the pip-egg is called pycoalaip, so as long as we use the dependency-links we'll have to go with that.